### PR TITLE
Fix RH Hybrid Cloud Console mentions

### PR DESCRIPTION
**Pull request by Lennonka**
cloud.redhat.com URL is a redirect to console.redhat.com, which can cause timeout errors (especially with API calls)

https://bugzilla.redhat.com/show_bug.cgi?id=2149457

* [x] I am familiar with the [contributing](https://github.com/theforeman/foreman-documentation/blob/master/CONTRIBUTING.md) guidelines.

Please cherry-pick my commits into:

* [x] Foreman 3.5/Katello 4.7 (planned Satellite 6.13)
* [x] Foreman 3.4/Katello 4.6
* [x] Foreman 3.3/Katello 4.5 (Satellite 6.12)
* [x] Foreman 3.2/Katello 4.4
* [x] Foreman 3.1/Katello 4.3 (Satellite 6.11, orcharhino 6.1+)
* For Foreman 3.0 or older, please create a separate PR.
* We do not accept PRs for Foreman 2.3 or older.

### CODE REVIEW

File: guides/common/modules/ref_ports-and-firewall-requirements.adoc [hunk 1]
```diff
@@ -94,8 +94,8 @@ Remote Execution result upload
 | 443 | TCP | HTTPS | Amazon EC2, Azure, Google GCE | Compute resources | Virtual machine interactions (query/create/destroy) (optional)
 ifdef::satellite[]
 ifeval::["{mode}" == "connected"]
-| 443 | TCP | HTTPS | cloud.redhat.com | Red{nbsp}Hat Cloud plugin API calls |
-| 443 | TCP | HTTPS | Red{nbsp}Hat Portal | SOS report | Assisting support cases (optional)
+| 443 | TCP | HTTPS | Red{nbsp}Hat Hybrid Cloud Console | Red{nbsp}Hat Cloud plugin API calls |
```
Comment (ekohl) [inline guides/common/modules/ref_ports-and-firewall-requirements.adoc:97]: If I'm writing firewall rules I prefer to have a service name. I can resolve `console.redhat.com` using DNS, I can't resolve a description.

Having said that: if it's on a CDN that may also be hard to figure out 100% reliably, but it at least gives a hint.

File: guides/common/modules/ref_ports-and-firewall-requirements.adoc [hunk 2]
```diff
@@ -94,8 +94,8 @@ Remote Execution result upload
...
+| 443 | TCP | HTTPS | Red{nbsp}Hat Customer Portal | SOS report | Assisting support cases (optional)
 | 443 | TCP | HTTPS | Red{nbsp}Hat CDN | Content Sync | Red{nbsp}Hat CDN
```
Comment (ekohl) [inline guides/common/modules/ref_ports-and-firewall-requirements.adoc:99]: FYI: there's an article on the IPs for the CDN: https://access.redhat.com/articles/1525183.

File: guides/common/modules/ref_ports-and-firewall-requirements.adoc (hunk 1 above)
Comment (Lennonka) [reply guides/common/modules/ref_ports-and-firewall-requirements.adoc:97]: Gotcha.

File: guides/common/modules/ref_ports-and-firewall-requirements.adoc (hunk 2 above)
Comment (Lennonka) [reply guides/common/modules/ref_ports-and-firewall-requirements.adoc:99]: It's mentioned in another module.

**Comment by Lennonka** [comment]
@ehelms I just replaced "cloud" with "console". The other suggestions are out of scope of this PR.

**Review by ekohl — APPROVED** [review]

File: guides/common/modules/proc_synchronizing-ansible-collections.adoc [hunk 3]
```diff
@@ -1,10 +1,7 @@
-:_module-type: PROCEDURE
-
 [id="proc_synchronizing-ansible-collections_{context}"]
```
Comment (maximiliankolb) [inline guides/common/modules/proc_synchronizing-ansible-collections.adoc:1]:
```suggestion
[id="synchronizing-ansible-collections_{context}"]
```

Unrelated to your PR; feel free to ignore.

File: guides/common/modules/proc_synchronizing-ansible-collections.adoc [hunk 4]
```diff
@@ -23,7 +20,7 @@ The *Label* field is populated automatically based on the name.
 . In the *Upstream URL* field, enter the URL for the upstream collections repository.
 +
 The URL can be any Galaxy endpoint.
-For example, `++https://cloud.redhat.com/api/automation-hub/++`.
+For example, `++https://console.redhat.com/api/automation-hub/++`.
```
Comment (maximiliankolb) [inline guides/common/modules/proc_synchronizing-ansible-collections.adoc:23]:
```suggestion
For example, `https://console.redhat.com/api/automation-hub/`.
```

`++` is required if links contain spaces or special characters: https://docs.asciidoctor.org/asciidoc/latest/syntax-quick-reference/#links

Here, it's IMHO not needed.

**Comment by Lennonka** [comment]
Cherry-picked:
- 9b0d94c2f..84a5bd094  3.5 -> 3.5
- 252dbed7c..a351fd5da  3.4 -> 3.4
- 7663ea301..0f6281418  3.3 -> 3.3
- 4e3467832..ffa0cac91  3.2 -> 3.2
- e25996372..6b385feae  3.1 -> 3.1

**Pull request merged**